ckan/public/base/scss/_bootstrap.scss

```diff
@@ -0,0 +1,56 @@
+/*!
+ * Bootstrap v3.3.7 (http://getbootstrap.com)
```

Comment:
This file is based on the wrong version, probably doesn't have any other differences than the version number but for consistency sake it should be 3.4.1